__tests__/app.test.js

```diff
@@ -261,6 +260,82 @@ describe('GET /api/reviews/:review_id/comments', () => {
     });
 });
 
+describe('POST /api/reviews/:review_id/comments', () => {
+    it('returns 201 status code with new comment when passed valid request', () => {
+        const newComment = {
+            username: "dav3rid",
+            body: "Good soldiers follow orders"
+        }
+
+        return request(app)
+        .post('/api/reviews/5/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+            const { comment } = body
+            expect(comment).toBeInstanceOf(Object)
+            expect(comment.comment_id).toBe(7)
+            expect(comment.votes).toBe(0)
+            expect(comment.author).toBe("dav3rid")
+            expect(comment.body).toBe("Good soldiers follow orders")
+            expect(comment.review_id).toBe(5)
+            expect(comment).toHaveProperty("created_at")
```

Comment:
This would be better done with expect.objectContaining

Reply:
While I agree that would potentially tidier and require less code, given I wanted to test that the correct value was at the correct key, this was more explicit and avoids any possibility of a false positive (i.e. if for some reason comment_id had come back as 5 and review_id had come back as 7).